tst/EnergyPlus/unit/CMakeLists.txt

```diff
@@ -272,9 +272,10 @@ set(test_src
     ZoneHVACEvaporativeCooler.unit.cc
     ZonePlenum.unit.cc
     ZoneTempPredictorCorrector.unit.cc
+    CommandLineInterface.unit.cc
     main.cc)
 
-set(test_dependencies energypluslib)
+set(test_dependencies energypluslib energyplusapi)
```

Comment:
I got rid of the separate energyplusapi_tests build.  Just build the unit tests into a single test executable that depends on the lib and the api both being built.  These are not optional things in the EnergyPlus build, so it should be fine for every platform, dev, machine, etc.  (Famous last words)